internal/scorecard/alpha/testdata/memcached-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
```

Comment:
We shouldn't be using package manifests any more as they are [deprecated](https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md#graph-metadata).

Reply:
I'll rework this as soon as the package manifest/configmap bug is resolved we talked about today.  Let me know if you think this is worth holding this PR up for.

Reply:
nevermind, that doesn't hold this up, I removed the package manifest from the scorecard test config since its deprecated going forward.  Thanks for pointing that out.